source/client.js

```diff
@@ -13,7 +15,11 @@ const pageDetails = parse(window.location.search)
 
 render(
   <div>
```

Comment:
Mixpanel has dependencies on `document` existing and renderToStaticMarkup is hitting MP code and failing